pset-iterators.js

```diff
@@ -61,8 +61,13 @@ declared with const
 ************/
 console.log('Problem 1:')
 
+
 // Add your code below this line
 
+const presidentsNamedJames = presidents.filter(kingJames => kingJames.president.indexOf('James') === 0)
```

Comment:
@Defy1x nice use of indexOf to match against the "James", alternative approaches involve using `.startsWith()` or `.substring()`